docs/guides/expectations/contributing/how-to-contribute-a-new-expectation-to-great-expectations.md

```diff
@@ -223,45 +249,129 @@ The Expectation in the template (expect_column_values_to_equal_three) did not ne
+````
+
+* Add the new arguments to the test cases in the examples.
+:::
+
+:::note
 
 Some Column Map Metrics that map every row of a column to yes/no need a numeric value pre-computed for each row in order to produce the answer.
 
-This requires defining a new Metric. The parent class of your Metric Provider class (ColumnMapMetricProvider) provides support for this case.
+This requires defining a new Metric. The parent class of your Metric Provider class (`ColumnMapMetricProvider`) provides support for this case.
 
-A good example of this pattern is expect_column_value_z_scores_to_be_less_than - one of the core Expectations.
+A good example of this pattern is `expect_column_value_z_scores_to_be_less_than` - one of the core Expectations.
 
-The Expectation declares “column_values.z_score.under_threshold” as its condition_metric_name (the Metric that answers the yes/no question for every row).
+The Expectation declares “column_values.z_score.under_threshold” as its `condition_metric_name` (the Metric that answers the yes/no question for every row).
 
-The ColumnValuesZScore Metric Provider class that computes this Metric declares an additonal metric:
+The `ColumnValuesZScore` Metric Provider class that computes this Metric declares an additonal metric:
 
+````console
 function_metric_name = "column_values.z_score"
-The class implements methods decorated with @column_function_partial to compute the Z score for every row for each backend.
+````
+
+The class implements methods decorated with `@column_function_partial` to compute the Z score for every row for each backend.
 
 Consult the following files for the details of this pattern:
 
-great_expectations/great_expectations/expectations/core/expect_column_value_z_scores_to_be_less_than.py
-great_expectations/great_expectations/expectations/metrics/column_map_metrics/column_values_z_score.py
-Fill in the ``library_metadata`` dictionary.
+* great_expectations/great_expectations/expectations/core/expect_column_value_z_scores_to_be_less_than.py
+* great_expectations/great_expectations/expectations/metrics/column_map_metrics/column_values_z_score.py
+:::
+
+
+##### ColumnExpectation
+Expectations that extend ColumnExpectation class are evaluated for a single column, but produce an aggregate metric, such as a mean, standard deviation, number of unique values, type, etc.
+
+Define success_keys of your Expectation
+
+success_keys = ("min_value", "strict_min", "max_value", "strict_max")
+
+Expectations rely on Metrics to produce their result. A Metric is any observable property of data (e.g., numeric stats like mean/median/mode of a column, but also richer properties of data, such as histogram). You can read more about the relationship between Expectations and Metrics in our [Core Concepts: Expectations and Metrics](docs/reference/metrics).
+
+* `ExpectColumnCustomMedianToBeBetween` class that the template implements declares the list of Metrics it needs computes for producing its result:
+
+````console
```

Comment:
I think any instance of `console` should be turned to `python` unless it is a `bash` output. Thoughts? Don't think it makes a big difference with one-liners but it could help clarify things as we iterate on these docs.

Reply:
I've resolved any other instances of this comment so it isn't overbearing. If you and any other reviewer could do a quick sweep of these for all files committed, that would be greatly appreciated!

Reply:
```suggestion
````python
```